### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by FlutterFraz** [Floorp-Projects/Floorp#1519]
### Pre-Submission Checklist

- [X] I have checked the existing/closed issues to ensure this issue has not already been reported.
- [X] I am using the latest supported version of the browser.
- [X] I have tried disabling all browser extensions/add-ons to rule out interference from these.
- [X] I have tried clearing the browser cache and cookies.
- [X] I have checked with a new profile and the issue still occurs.
- [X] I have provided detailed step-by-step instructions on how to reproduce the issue.
- [ ] I have included relevant screenshots or console outputs.
- [X] I have checked that this problem is not occurring in the latest version of Firefox ESR.
- [X] This issue is specific to this browser and does not occur in other browsers.

### Bug Summary

Simply put, from the last update, 11.20.0(64 bit) | Firefox128.5.0, all my profiles suffer from this. All windows, tabs and workspaces get deleted after closing. Only performing a reboot from the burger menu saves them.

### Reproduction Steps

1. Have some tabs and/or windows and/or workspaces open
2. Close the browser
3. Reopen the browser

### Expected Behavior

Tabs, windows and workspaces still present

### Actual Behavior

Tabs, windows and workspaces not present

### Screenshots

![immagine](https://github.com/user-attachments/assets/23cbc9cc-5bb3-4d2a-8c49-1bbbdde9d7a6)

### Browser Version

11.20.0(64 bit)

### Operating System

Windows 11 Home 23H2

**Comment by FlutterFraz** [comment]
I realized that if I have two Windows desktops active, a new window will pop open in the second desktop and the main one will be in the first. Not sure if this means its not a bug at all.

**Comment by melroy89** [comment]
> Floorp 11's Workspaces are saved per window, so closing a window also closes the Workspaces. These may be restored via History > Recently closed windows.

Sorry but I had 3 workspaces with each workspace like 6, 7 or 8 tabs. And after an update .. I lost _everything_.. So its still not fixed ;(

**Comment by mouarfff** [comment]
Same issue here, 
BUG STILL NOT FIXED

2000 tabs to retrieve from the profil management with no any easy way to select all tabs to pu in my worpsaces....

Tested with flatpak on Bazzite and Linux Mint 22.3 XFCE

the first i lost all my tabs in bazzite floorp after cloe the LAST tab of a workspace

and i just stay my another system and find my another flloorp (on mint wfce) EMPTY ....

SO 👍 loose your tabs in a profil in a pc can lost your tab in ALL OTHER FLOORP INSTALLATION SYNCRONISED

I hade very very important things

first time since 20 years i loose browser tabs.....
